bin/update

```diff
@@ -40,17 +41,27 @@ $execute = function(array $command, $timeout = 60) use ($output) {
         ->run($output)
     ;
 };
-$executeAndCheck = function(array $command, $timeout = 60) use ($output) {
-    ProcessBuilder::create()
-        ->setArguments($command)
-        ->getProcess()
-        ->setTimeout($timeout)
-        ->enableOutput()
-        ->mustRun($output)
-    ;
+$executeAndCheck = function(array $command, $timeout = 60, $attempts = 1) use ($output) {
+    for ($i = 1; $i <= $attempts; $i++) {
+        try {
+            ProcessBuilder::create()
```

Comment:
Wouldn't this then also run twice if it succeeds?

Reply:
Right, it's idempotent but no need for it